fiftyone/utils/torch.py

```diff
@@ -251,32 +252,50 @@ class TorchEmbeddingsMixin(fom.EmbeddingsMixin):
         layer_name (None): the name of the embeddings layer whose output to
             save, or ``None`` if this model instance should not expose
             embeddings. Prepend ``"<"`` to save the input tensor instead
+        feature_extractor (False): whether this model instance should
```

Comment:
since this is a bool, can you rename this to `extract_features` or `as_feature_extractor` to make it more clear

Reply:
renamed!